Pipfile.lock

```diff
@@ -80,13 +94,6 @@
             "index": "pypi",
             "version": "==6.7"
         },
-        "configparser": {
```

Comment:
@The-Compiler hmmm, apperently `pipenv` doesn't like me. Can you recreate the lock?